### PR TITLE
Add `Wiki maintainers` page

### PR DESCRIPTION
**Pull request by BlueChinchompa**
This will be useful for the How to contribute to the titanic wiki page in the wiki.

### CODE REVIEW

File: wiki/People/Wiki_Maintainers/en.md
Comment (iSlodinxOsu) [inline wiki/People/Wiki_Maintainers/en.md:2]:
```suggestion
Wiki Maintainers are a group of users who are in charge of maintaing the wiki and merging changes into the official Titanic! wiki.
```

File: wiki/People/Wiki_Maintainers/en.md
Comment (iSlodinxOsu) [inline wiki/People/Wiki_Maintainers/en.md:11]: Levi should know about this info, it would be best to wait for review

File: wiki/People/Wiki_Maintainers/en.md
Comment (iSlodinxOsu) [inline wiki/People/Wiki_Maintainers/en.md:12]:
```suggestion
[iSlodinx](https://osu.titanic.sh/u/869) | October 19th, 2024
```

File: wiki/People/Wiki_Maintainers/en.md
Comment (iSlodinxOsu) [inline wiki/People/Wiki_Maintainers/en.md:13]:
```suggestion
[Meru](https://osu.titanic.sh/u/41) | October 19th, 2024
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

File: wiki/People/Wiki_Maintainers/en.md
Comment (iSlodinxOsu) [inline wiki/People/Wiki_Maintainers/en.md:1]:
```suggestion
## Wiki maintainers
```

File: wiki/People/Wiki_Maintainers/en.md
Comment (iSlodinxOsu) [inline wiki/People/Wiki_Maintainers/en.md:4]:
```suggestion
## Roles and responsibilities
```

File: wiki/People/Wiki_Maintainers/en.md
Comment (iSlodinxOsu) [inline wiki/People/Wiki_Maintainers/en.md:5]:
```suggestion
Wiki maintainers are responsible for reviewing and merge pull requests to make sure there are no errors.
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]
Add lowercase on .md file since it says `Wiki_Maintainers` but it needs to be `Wiki_maintainers`.

File: wiki/People/Wiki_maintainers/en.md
Comment (iSlodinxOsu) [inline wiki/People/Wiki_maintainers/en.md:7]:
```suggestion
## Current members
```

File: wiki/People/Wiki_maintainers/en.md
Comment (iSlodinxOsu) [inline wiki/People/Wiki_maintainers/en.md:11]:
```suggestion
[BlueChinchompa](https://osu.titanic.sh/u/40) | Since The Beginning 
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]
im blind...

File: wiki/People/Wiki_Maintainers/en.md
Comment (BlueChinchompa) [reply wiki/People/Wiki_Maintainers/en.md]: I agree I tried searching around in the `#Wiki` channel in discord and any mention of maintainer/invite but I didn't find anything.

Comment (iSlodinxOsu) [reply wiki/People/Wiki_Maintainers/en.md]: oh wait, you may find it on your email address which is connected to your github, perhaps try to find it on gmail or something

File: wiki/People/Wiki_maintainers/en.md
Comment (Lekuruu) [inline wiki/People/Wiki_maintainers/en.md:5]:
```suggestion
Wiki maintainers are a group of users who are in charge of maintaining the wiki and merging changes into the official Titanic! wiki. They are responsible for reviewing and merge pull requests to make sure there are no errors.
```
What do you think about this?

File: wiki/People/Wiki_Maintainers/en.md
Comment (Lekuruu) [reply wiki/People/Wiki_Maintainers/en.md]: It would be on your gmail, yeah

File: wiki/People/Wiki_maintainers/en.md
Comment (iSlodinxOsu) [reply wiki/People/Wiki_maintainers/en.md:5]: looks better in that way though.

Comment (BlueChinchompa) [reply wiki/People/Wiki_maintainers/en.md:5]: Sounds way better (What the fuck was I doing at like 1 am)

**Review by iSlodinxOsu — APPROVED** [review]
it's good to go.

**Comment by BlueChinchompa** [comment]
> it's good to go.

should we still try to find the date I joined or just fuck it

**Comment by iSlodinxOsu** [comment]
> > it's good to go.
> 
> should we still try to find the date I joined or just fuck it

if you still can manage to find it, make a change

**Comment by BlueChinchompa** [comment]
Okay this should be good to merge now. I don't remember if Digital became on yet or not.

**Review by iSlodinxOsu — APPROVED** [review]

**Review by FreddyBadAdd — APPROVED** [review]
it has me so approving

**Pull request merged**
